test/inductor/test_mkldnn_pattern_matcher.py

```diff
@@ -1628,6 +1629,18 @@ def test_qlinear_cpu(self):
         for bias in [True, False]:
             self._qlinear_cpu_test_helper((torch.randn((2, 4)),), bias=bias)
 
+    @skipIfNoDynamoSupport
+    @skipIfNoONEDNN
+    @skipIfNoXPU
+    def test_qlinear_xpu(self):
+        r"""
+        This testcase will quantize a single Linear Moduel.
+        """
+        for bias in [True, False]:
+            self._qlinear_cpu_test_helper(
```

Comment:
In addition, `_qlinear_cpu_test_helper ` only tests fused pattern matcher count as it sets `check_quantization` to `True`. Is it necessary to add some UTs to test the accuracy of qliner we added in this PR?